scripts/DocGenerator.php

```diff
@@ -96,13 +96,20 @@ private function buildDocument($reflector)
         $parts = explode('_', get_class($reflector->getNode()));
         $type = end($parts);
 
+        $docBlock = $reflector->getDocBlock();
+        $methods = $reflector->getMethods();
+
+        if (is_null($docBlock)) {
```

Comment:
@dwsupplee I added some debugging information. Nothing fancy, but gives you a hint of where the doc script is failing. Would be nice to take another look at this when we have some time.